.github/workflows/metadata-test.yml

```diff
@@ -22,8 +26,7 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - 3.6
-          - 3.7
+          - 3.12
```

Comment:
Python 3.10 is not working here?

To make it more in line with GitHub actions you could use the action like we do it here:

https://github.com/OpenEnergyPlatform/omi/blob/6de5bb14d094b868680bb465803f9d410e283803/.github/workflows/pypi-publish.yml#L13-L17